README.md

```diff
@@ -7,7 +7,7 @@
 Add pcd-rs to your `Cargo.toml`.
 
 ```toml
-pcd_rs = "0.8.0"
+pcd-rs = "0.10.0"
```

Comment:
pcd_rs -> pcd-rs because the dependency name in Cargo.toml must exactly match the published crate name.
Otherwise, you will get the following error:

```
error: no matching package found
searched package name: `pcd_rs`
perhaps you meant:      pcd-rs
location searched: registry `crates-io`
```